apps/chat/src/store/chat/chat.selectors.ts

```diff
@@ -0,0 +1,35 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+import { ChatState } from '@/src/store/chat/chat.reducer';
+import { ConversationsSelectors } from '@/src/store/conversations/conversations.reducers';
+
+import { RootState } from '@/src/store';
+
+const rootSelector = (state: RootState): ChatState => state.chat;
+
+export const selectInputContent = createSelector(
+  [rootSelector],
+  (state) => state.inputContent,
+);
+
+export const selectChatFormOptions = createSelector(
+  [rootSelector],
+  (state) => state.formOptions,
+);
+
+export const selectIsChatInputDisabled = createSelector(
```

Comment:
I think it's better to pass selected chat as a param. Importing another selectors may unexpectedly lead to a circular dependency.